isthmus/src/test/java/io/substrait/isthmus/sql/SubstraitCreateStatementParserTest.java

```diff
@@ -0,0 +1,118 @@
+package io.substrait.isthmus.sql;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.apache.calcite.prepare.CalciteCatalogReader;
+import org.apache.calcite.sql.parser.SqlParseException;
+import org.junit.jupiter.api.Test;
+
+class SubstraitCreateStatementParserTest {
+  @Test
+  void testProcessCreateStatementsToCatalogWithSingleCreateTableWithTableNameOnly()
+      throws SqlParseException {
+    final CalciteCatalogReader catalogReader =
+        SubstraitCreateStatementParser.processCreateStatementsToCatalog(
+            "create table src1 (intcol int, charcol varchar(10))");
+
+    assertEquals(catalogReader.getRootSchema().getTableNames().size(), 1);
```

Comment:
Nothing functionally wrong with it but `catalogReader.getRootSchema().` gets repeated  in (almost) every assertion throughout the file. It might be worth pulling out as a variable so each assertion is easier to read.

```suggestion
    CalciteSchema schema = catalogReader.getRootSchema();
    assertEquals(schema.getTableNames().size(), 1);
```